requirements/linux/all-requirements.txt

```diff
@@ -2,23 +2,23 @@
 # Please do not edit it manually.
 
 aicscytoparam==0.1.12
-aicsimageio==4.14.0
+aicsimageio==4.11.0
```

Comment:
tifffile was upgraded but aicsimageio was downgraded? Also do we install both bioio and aicsimageio?

Reply:
cyto-dl just uses bioio, aicsimageio is a dependency of pyshtools